src/javascript/box.js

```diff
@@ -0,0 +1,108 @@
+import API from './api.js';
+
+const api = new API();
+const numberWithCommas = s => String(s).replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+
+export default function Box({ target, buttonTarget, page = 1, count = 5 }) {
+  this.target = target;
+  this.buttonTarget = buttonTarget;
+  this.page = page;
+  this.count = count;
+  this.boxLength = 0;
+  this.boxPrefix = 'box__item';
+  this.buttonPrefix = 'more-button';
+  this.countTarget = null;
+}
+
+Box.prototype = {
+  init,
+  getBoxLength,
+  getBoxList,
+  renderBoxList,
+  getBoxBlock,
+  renderMoreButton,
+  isMoreItem,
+  setDisableMoreButton,
+  updateCount,
+  createMoreEvent,
+}
+
+Box.constructor = Box;
+
+async function init() {
+  this.boxLength = await this.getBoxLength();
+  this.renderMoreButton();
+  await this.renderBoxList();
+  this.createMoreEvent();
+}
+
+async function renderBoxList() {
+  const data = await this.getBoxList();
```

Comment:
this context를 사용하지 않는 메서드는 사용하지 않는 것이 좋겠습니다. 호출되는 모습만 보면 this context를 사용할 것이라 유추하게 되겠죠?
```suggestion
  const data = await api.getItem({ type: 'box', page: this.page, count: this.count});
```